gridstatus/tests/test_isos.py

```diff
@@ -1,52 +1,13 @@
 import pandas as pd
```

Comment:
I think we can rename this file now that is just the misc tests that remain. perhaps `test_gridstatus.py`?

perhaps also just leave a comment that ISO general tests go in `base_test_iso` and ISO specific test go in their respective files, so it clear what goes here.